python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py

```diff
@@ -131,3 +131,43 @@ def get_scheduler(self) -> typing.Tuple[int, CoreScheduler]:
         scheduler = CoreScheduler(self.surveys, nside=self.nside)
 
         return self.nside, scheduler
+
+
+def without_keys(d: dict, keys: typing.List[str]) -> dict:
```

Comment:
Do you really need this method at al? it is a one-liner. Just do it inline?

Reply:
Going with inline option